docs/source/develop.rst

```diff
@@ -91,7 +91,8 @@ Install dask and dependencies::
 
 For development dask uses the following additional dependencies::
 
-   pip install pytest moto mock
+   pip install pytest moto mock partd sparse
+   conda install cloudpickle
```

Comment:
cloudpickle and partd should be included in `dask[complete]`.  Sparse should be entirely optional.  If there are failing tests then I would rather skip them if sparse isn't installed than require that people install sparse.